src/VisualStudio/Core/Def/xlf/ServicesVSResources.cs.xlf

```diff
@@ -209,37 +209,42 @@
       </trans-unit>
       <trans-unit id="Select_destination">
         <source>Select destination</source>
-	<target state="new">Select destination</target>
+        <target state="new">Select destination</target>
         <note />
       </trans-unit>
       <trans-unit id="Select_Dependents">
         <source>_Select Dependents</source>
-	<target state="new">_Select Dependents</target>
+        <target state="new">_Select Dependents</target>
         <note />
       </trans-unit>
       <trans-unit id="Select_Public">
         <source>_Select Public</source>
-	<target state="new">_Select Public</target>
+        <target state="new">_Select Public</target>
         <note />
       </trans-unit>
       <trans-unit id="Select_destination_and_members_to_pull_up">
         <source>Select destination and members to pull up.</source>
-	<target state="new">Select destination and members to pull up.</target>
+        <target state="new">Select destination and members to pull up.</target>
```

Comment:
@CyrusNajmabadi @tmeschter Why does this happen? Do we need to worry about these whitespace changes?

Reply:
I don't knw.  Looks like my vs decided to update tabs/spaces.  I'm on:

![image](https://user-images.githubusercontent.com/4564579/53533767-f294bd80-3ab0-11e9-83e9-c532bc8ef25d.png)